services/hubble-port-forward.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Hubble port-forward service
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+User=ubuntu
+ExecStart=cilium hubble port-forward
```

Comment:
which port is this exposing?

Reply:
It connects to the `hubble-relay` service on the `4245` port. It seems to be the default method: https://docs.cilium.io/en/stable/gettingstarted/hubble_setup/#hubble-setup